<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorWidget</class>
 <widget class="QDockWidget" name="InspectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>494</height>
   </rect>
  </property>
  <property name="toolTip">
   <string>Inspector panel</string>
  </property>
  <property name="whatsThis">
   <string>Element or Connection Inspector panel</string>
  </property>
  <property name="allowedAreas">
   <set>Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Inspector</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="propertiesTab">
       <attribute name="title">
        <string>Properties</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </widget>
      <widget class="QWidget" name="infoTab">
       <attribute name="title">
        <string>Information</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextBrowser" name="infoBox">
          <property name="styleSheet">
           <string notr="true">background: transparent;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusMsg">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
